components/esp_websocket_client/examples/target/main/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
 idf_component_register(SRCS "websocket_example.c"
-                    INCLUDE_DIRS ".")
+                    INCLUDE_DIRS "."
+                    EMBED_TXTFILES "certs/client_cert.pem"
```

Comment:
Are we embedding these file unconditionally? Even if they are n ot used unde the default config?